docs/docsite/rst/os_guide/windows_performance.rst

```diff
@@ -20,24 +20,50 @@ seconds.
 
 .. code-block:: powershell
 
-    function Optimize-PowershellAssemblies {
-      # NGEN powershell assembly, improves startup time of powershell by 10x
-      $old_path = $env:path
-      try {
-        $env:path = [Runtime.InteropServices.RuntimeEnvironment]::GetRuntimeDirectory()
-        [AppDomain]::CurrentDomain.GetAssemblies() | % {
-          if (! $_.location) {continue}
-          $Name = Split-Path $_.location -leaf
-          if ($Name.startswith("Microsoft.PowerShell.")) {
-            Write-Progress -Activity "Native Image Installation" -Status "$name"
-            ngen install $_.location | % {"`t$_"}
-          }
-        }
-      } finally {
-        $env:path = $old_path
-      }
+    function Optimize-Assemblies {
+        param (
+            [string]$assemblyFilter = "Microsoft.PowerShell.",
+            [string]$activity = "Native Image Installation"
```

Comment:
```suggestion
            [string]$AssemblyFilter = "Microsoft.PowerShell.",
            [string]$Activity = "Native Image Installation"
```

Reply:
This is an argument between CamelCase and PascalCase. The most widely accepted case for PowerShell is CamelCase.